### PR TITLE
feat: fix functions names & related terminology to be imperative

### PR DESCRIPTION
**Pull request by getsnoopy**
## What does this pull request change?
This commit changes the `login` function to be `logIn`. It also updates any related terminology in the docs or otherwise to match.

## Why is this pull request needed?
Function names should conventionally be in the imperative form, but the login() function didn't conform to this convention.

## Issues related to this change
#115

### CODE REVIEW

File: src/Types.ts [hunk 1]
```diff
@@ -79,7 +80,7 @@ export type TAuthConfig = {
 }
 
 export type TRefreshTokenExpiredEvent = {
-  login: () => void
+  logIn: () => void
```
[the remark this answers is not in the document]
Comment (sebastianvitterso) [reply src/Types.ts:83]: I do appreciate this, however
image

So just add the `login` as a deprecated property to the type, and we'll be there.

**Review by sebastianvitterso — CHANGES_REQUESTED** [review]
Almost perfect! Thanks for the PR, just a few notes and I think we can merge it!

**Comment by soofstad** [comment]
Thanks for the PR, I have nothing to add beyond Sebastians review.
Let's fixed that, and it's ready for merge :slightly_smiling_face:

**Comment by getsnoopy** [comment]
Yeah; just been busy with work and life. I can get to this next week.

**Comment by getsnoopy** [comment]
Apologies for the delay. I've updated the code now.

**Review by sebastianvitterso — APPROVED** [review]
Nice, thanks!

**Review by soofstad — APPROVED** [review]

**Pull request merged**